charts/tempo-distributed/values.yaml

```diff
@@ -43,7 +43,7 @@ externalRuntimeConfigName: '{{ include "tempo.resourceName" (dict "ctx" . "compo
 externalConfigVersion: '0'
 
 # -- If true, Tempo will report anonymous usage data about the shape of a deployment to Grafana Labs
-reportingEnabled: true
```

Comment:
I think we would like to leave this.  Feel free to call something out more explicit in the docs if you'd like.

Reply:
Sure, the comments seems quite straight forward (if you have something else in mind please let me know)